libs/sdk-ui-ext/src/internal/constants/supportedProperties.ts

```diff
@@ -76,9 +77,13 @@ export const COLUMN_CHART_SUPPORTED_PROPERTIES = {
 };
 
 export const LINE_CHART_SUPPORTED_PROPERTIES = {
-    [AXIS.PRIMARY]: BASE_CHART_SUPPORTED_PROPERTIES,
+    [AXIS.PRIMARY]: [...BASE_CHART_SUPPORTED_PROPERTIES, "dataPoints.visible"],
     [AXIS.SECONDARY]: [...BASE_PROPERTIES, ...BASE_X_AXIS_PROPERTIES, ...BASE_SECONDARY_AXIS_PROPERTIES],
```

Comment:
why is not the new control also on secondary axis?

Reply:
That's actually why I need to consult this. I was following how dataLabels were implemented and assumed primary axis is for the y axis? But I guess I assumed wrong.